src/components/ebay-calendar/index.marko

```diff
@@ -64,37 +69,33 @@ $ const rangeEndText = input.a11yRangeEndText ?? "end of range"
                                     $ const isInRange = component.isInRange(dayISO);
                                     $ const isRangeEnd = dayISO === state.rangeEnd;
-                                    $ const disabled = component.isDisabled(dayISO);
-                                    $ const rangeAriaLabel = (input.range && Array.isArray(input.selected) && (
-                                            isRangeStart
-                                            ? `${day} - ${rangeStartText}`
-                                            : isRangeEnd
-                                            ? `${day} - ${rangeEndText}`
-                                            : isInRange
-                                            ? `${day} - ${inRangeText}`
-                                            : undefined
-                                        ))
-                                    <td
-                                        class={
+                                    $ const isDisabled = component.isDisabled(dayISO);
+                                    $ let a11yTexts = [""];
+                                    $ if (isSelected) a11yTexts.push(a11ySelectedText);
+                                    $ if (isRangeStart) a11yTexts.push(a11yRangeStartText);
+                                    $ if (!isRangeStart && !isRangeEnd && isInRange) a11yTexts.push(a11yInRangeText);
+                                    $ if (isRangeEnd) a11yTexts.push(a11yRangeEndText);
+
+                                    <td class={
+                                            "calendar__cell--selected": isSelected,
                                             "calendar__range--start": isRangeStart,
                                             "calendar__range": isInRange,
                                             "calendar__range--end": isRangeEnd,
-                                        }
-                                        aria-current=dayISO === state.todayISO && "date"
-                                        aria-selected=(
-                                            Array.isArray(input.selected)
-                                                ? input.selected.some((iso) => iso === dayISO)
-                                                : input.selected && input.selected === dayISO
-                                        ) && "true">
+                                        }>
                                         <if(input.interactive)>
                                             <button
                                                 key=dayISO
-                                                disabled=disabled
-                                                aria-label=rangeAriaLabel
+                                                disabled=isDisabled
+                                                aria-label=(a11yTexts.length > 1) && `${day}${a11yTexts.join(a11ySeparator)}`
```

Comment:
Does this need a separator or something between day and a11yText's join?

Reply:
No separator is needed, because the array starts as a single empty string so effectively it begins with a separator